manual_step/README

```diff
@@ -0,0 +1,25 @@
+# Ideas
+
+## Entry in jupyter notebook cell
+* Markup is generated in the cell corresponding to the RDF description of the manual step.
+* A "results" input section of the cell allows user to input URI of expected/resultant data sets.
```

Comment:
I feel like handling a manual step in the jupyter notebook might leave a lot of room for mistakes and typos. A simple web form with some input fields might also be an option. I feel though that you have a strong preference for using jupyter notebooks for manual steps. What is your reasoning?

Reply:
The frontend of a notebook extension is just javascript, so you can have a web form or whatever UI features you want, for the manual step. It's more about keeping things in the same place so people don't have to load up a different website/window for different steps in the same workflow. So, for example, each cell in the notebook would know whether it was a computational step (normal python code), or a manual step (displayed using whatever UI features we choose). We can then "easily" save or publish the workflow and individual steps all in one go.

Reply:
Sounds good. It's nice that we can be so flexible with our jupyer notebook extension.